pkg/services/settings/settings.go

```diff
@@ -0,0 +1,43 @@
+package settings
```

Comment:
I tried to keep the new interface as similar as possible to the already existing one to reduce the required code changes as much as possible, but I'm not sure if there's an a good alternative that may worth it.